### PR TITLE
minor fixes + replace all usage of GEF Elf object with pyelftools

### PR DESCRIPTION
**Pull request by caprinux**
works on 64 bit and 32 bit binaries, still require some work to fully port from GEF Elf class to pyelftools

### CODE REVIEW

**Comment by caprinux** [comment]
addresses #24

File: decomp2gef.py [hunk 1]
```diff
@@ -362,37 +368,46 @@ def _construct_small_elf(self):
...
+        elf_data = self._force_update_text_size(stream, elf, elf_data, 0xFFFFFF)
 
         # find the symbol table
-        for section in elf.shdrs:
-            if section.sh_name == ".symtab":
-                break
-        else:
-            return
+        section = elf.get_section_by_name('.symtab')
+        if not section: return
```
Comment (mahaloz) [inline decomp2gef.py:401]:
```suggestion
        if not section: 
            return 
```

File: decomp2gef.py [hunk 2]
```diff
@@ -362,37 +368,46 @@ def _construct_small_elf(self):
...
-        else:
-            return
+        section = elf.get_section_by_name('.symtab')
+        if not section: return 
 
         # locate the location of the symbols size in the symtab
-        tab_offset = section.sh_offset
-        sym_data_size = 24 if elf.ELF_64_BITS else 16
+        tab_offset = section['sh_offset']
+        """
```
Comment (mahaloz) [inline decomp2gef.py:405]: comments in the middle of functions should all be "#" instead of the multiline comment style.

File: decomp2gef.py [hunk 3]
```diff
@@ -602,7 +617,7 @@ def title(self):
         if self.ready_to_display:
             title = "decompiler:{:s}:{:s}:{:d}".format(self.decompiler.name, self.curr_func, self.curr_line+1)
         else:
-            title = "decomipler:{:s}:error".format(self.decompiler.name)
+            title = "decompuiler:{:s}:error".format(self.decompiler.name)
```
Comment (mahaloz) [inline decomp2gef.py:620]:
```suggestion
            title = "decompiler:{:s}:error".format(self.decompiler.name)
```
Lol you placed a 'u' in decompiler

**Review by mahaloz — CHANGES_REQUESTED** [review]
This looks really good @caprinux. Few changes, and I need to verify the binaries on my machine as well, then I think we are good to go. Nice stuff.

**Review by mahaloz — APPROVED** [review]

**Pull request merged**